### ISSUE TITLE
BUG: When using the vanilla function to Copy a piece the rotation is blocked and alignment is wrong.

### ISSUE THREAD

**Issue by ZenDragonX** [JereKuusela/valheim-dev#56]
This is a problem for a mod I am working on.  The piece is not matching the original rotation.  I narrowed it down to a vanilla test with only ServerDevCommands running and you can not get a clean rotation match between the original piece and the cloned piece.  The issue is persistent even on gamepad.  

I looked through your code and it seems that you are preventing rotation, this is a problem.  I'm not sure what this feature is for.  How do I turn this off?

I am not entirely clear under what conditions a MouseWheelBinding is triggered in your code.  However, in my config file I have edit it to set to None yet it seems to always overwrite it with blank.
```

## The simulated key code when scrolling the wheel.
# Setting type: KeyboardShortcut
# Default value: None
Mouse wheel bind key = 
```

```

[HarmonyPatch(typeof(Player), nameof(Player.UpdatePlacement))]
public class PreventRotation
{
  static void Prefix(Player __instance, ref int __state)
  {
    __state = __instance.m_placeRotation;
  }
  static void Postfix(Player __instance, int __state)
  {
    if (MouseWheelBinding.CouldExecute())
      __instance.m_placeRotation = __state;
  }
}



```

**Comment by JereKuusela** [comment]
Empty is ok, that's how it interprets the none.

If you can select a build piece and rotate it with mouse wheel, then it should be ok.

But if you do for example `bind wheel say hello` then you can no longer rotate pieces because the command is executed every time instead.

**Comment by ZenDragonX** [comment]
The issue isn't that I can not rotate pieces, I can.  The issue is that when you copy a piece the new piece does not match the old piece rotation.  You are overwriting the m_placeRotation value.  Try it out.  Copy a piece and notice that the rotation is not matching the original piece that you copied.  Instead it's set to whatever your PRIOR piece rotation was of the last piece you built.  This is because when the CopyPiece method runs it sets the m_placeRotation to the piece you copied, however your mod is capturing the rotation of the cursor before PlacePiece runs (in prefix) and then overwriting the newly Cloned piece's rotation when you restore it in Postfix.  How can I turn this feature off? It's causing bugs for me.

**Comment by JereKuusela** [comment]
Ok, I understand now.

You probably have Infinity Tools installed that adds binding for mouse wheel + shift.

So when you have shift down, CouldExecute triggers.

I can fix this today or tomorrow, but now workaround is to change the Infinity Tools bind.

**Comment by ZenDragonX** [comment]
I don’t use infinity tools. This bug happens if your mod is installed by itself.

**Comment by JereKuusela** [comment]
You don't have to use workaround if you don't want to.

Anyways, this should be now fixed in version v1.78

**Comment by ZenDragonX** [comment]
Thanks, fixed.
